### PR TITLE
[Ready to Merge] make code cells not ugly

### PR DESCRIPTION
**Pull request by hamelsmu**
- made text sharper
- changed fonts
- dark theme for code, mirrors Darcula as much as possible
- Change spacing
- Made non-heading font color slightly lighter to provide an offset, and also increased the font size for readability
- Top padding on Headings

## Before 

image

## After

image

### CODE REVIEW

File: assets/main.scss [hunk 1]
```diff
@@ -28,20 +28,25 @@
...
-        margin-top:0.1rem !important;
+        margin-bottom:2rem !important;
+        margin-top:1.5rem !important;
         padding-bottom:0 !important;
         padding-top:0 !important;
+        background: #323443 !important;
+        -webkit-font-smoothing: antialiased;
+        text-rendering: optimizeLegibility;
+        font-family: BlinkMacSystemFont,'Segoe UI',Roboto,Ubuntu,'Helvetica Neue', sans-serif;
+        font-family: Menlo, Monaco, "Courier New", monospace;
```
Comment (jph00) [inline assets/main.scss]: I think you need less ugly Windows fonts in the list, ahead of Courier! :) e.g. Consolas and Lucida Console

File: assets/main.scss [hunk 2]
```diff
@@ -50,13 +55,74 @@
...
+.highlight .kc {
+        color: rgb(181, 206, 168);;
+}
+
+.code_cell table { width: auto; }
+
+.page-content {
+        -webkit-font-smoothing: antialiased !important;
+        text-rendering: optimizeLegibility !important;
+        font-family: "Segoe UI", "Open Sans", "Helvetica Neue";
```
Comment (jph00) [inline assets/main.scss]: Same here

File: assets/main.scss (hunk 1 above)
Comment (hamelsmu) [reply assets/main.scss]: Thank you so much, this is something I really struggle with and really appreciate the suggestions!   When it comes to aesthetics I am very "challenged" 😢  Will make the change 🙇

**Pull request merged**